torch/_inductor/codecache.py

```diff
@@ -292,8 +293,18 @@ def check_cache(cache, callback=None) -> bool:
                 and check_cache(self.get_global_cache(), callback=log_stats)
             ):
                 try:
-                    # re-benchmark everything to try to get consistent numbers from the same machine
```

Comment:
This comment seems important.  I think we do want consistent numbers from the same machine.  If you mix old numbers with new numbers it may not be apples-to-apples.

Typically, compiling is *much* more expensive than running.  So the cost of re-running (with a cached compile) should be very low.

Reply:
I will change that back if you like. It will literally mean that we run the same code again on the same machine, just at a different point in time. So I can imagine there is a difference if GPU / CPU power settings or background load have changed in the meantime, but not really much else.

Reply:
done